README.md

```diff
@@ -1,27 +1,58 @@
 # ![nf-core](nf-core-logos/nf-core-logo.png)
-# http://nf-co.re
 
-## [nf-core logos](nf-core-logos)
-- Countains original logo and icon in `ai` format, exported to `png` and also converted to `svg`
+## Repository contents
 
-## [make_logo](make_logo)
-- Countains a template logo with `GenericName` for a new pipeline, and a minimalist bash script to generate a new logo
+### [nf-core-logos/](nf-core-logos)
 
-### Requirements
-- Inkscape
-- [Maven Pro Bold](https://fonts.google.com/specimen/Maven+Pro) font.
+- Contains original logo and icon in `ai` format, exported to `png` and also converted to `svg`
+
+### [make_logo/](make_logo)
+
+- Contains a template logo with `GenericName` for a new pipeline, and a minimalist bash script to generate a new logo
+
+## Create logo using Docker
+
+We provide a Docker Image on [DockerHub](https://cloud.docker.com/u/nfcore/repository/docker/nfcore/logos) that you may use to create logos for your pipeline easily.
 
-Example:
+```bash
+docker run -v /path/on/host:/data -it nfcore/logos bash
+/make_logo.sh NewPipeline
+mv NewPipeline* /data
```

Comment:
If used with the `ENTRYPOINT` suggested above, I think this can be simplified to:

```bash
docker run -v `pwd`:`pwd` -w `pwd` nfcore/logos NewPipeline
```

Reply:
I guess we could add that in #8?  @drpatelh 

Also, the SIngularity image doesn't work as the current bash script basically writes where the Generic Logo currently resides at - which we should probably change to be configurable.